### ISSUE TITLE
Is an AppImage forthcoming?

### ISSUE THREAD

**Issue by johnblommers** [bagaswastu/papirus#1]
Would papirus be available as an AppImage at some point?

I really like the idea of an open source WYSIWYG Markdown editor by I'm not inclined to install another framework like appwrite to try out papirus.

**Comment by bgwastu** [comment]
Appimage? Do you mean Linux Appimage format? I think it's not possible since Papirus was a web app, not a desktop app.

Technically, it's possible to self-host Papirus on your machine. But you'll need Appwrite running on your device.

I know it's not an elegant way to self-host the app, and in the future, I plan to make the self-hosted version of Papirus, using SQLite as a database and packed on container images for easy deployment.

Just stay tuned for the upcoming updates!

**Comment by johnblommers** [comment]
Yes I mean the Linux AppImage format. The [appnativefy](https://ubuntuhandbook.org/index.php/2021/02/appnativefy-turn-website-executable-appimage/)  tool can wrap up a URL and create an AppImage. Of course any backend dependencies of the webapp would still have to be available on the Internet. So Papirus as an AppImage would not really make it standalone.

I look forward to a standalone version of Papirus.
